mangaki/mangaki/admin.py

```diff
@@ -358,6 +390,19 @@ def refresh_work_from_anidb(self, request, queryset):
 
     def merge(self, request, queryset):
         nb_merged, final_work, response = merge_works(request, queryset)
+        if response == 'no ID':
```

Comment:
We could make use of Enums, for example:

`response == MergeErrors.NO_ID` where `MergeErrors.NO_ID = 'no ID'`.

This way, it makes it easier to change the string value while keeping the functionality.